src/components/frontend-engine/yup/helper.ts

```diff
@@ -122,6 +122,7 @@ export namespace YupHelper {
 				case !!rule.positive:
 				case !!rule.negative:
 				case !!rule.integer:
+				case !!rule.unitNumberFormat:
```

Comment:
this doesn't need to be added because the validation has already been applied in `unit-number-field.tsx`
the rules handled here are for the inbuilt Yup rules only